src/app/dashboard/components/personal-achievements/personal-achievements.component.ts

```diff
@@ -1,15 +1,26 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
```

Comment:
you do not implement OnInit here, so pleasse remove it from import

Reply:
fixed